qdrant/grpc_client.go

```diff
@@ -39,6 +39,7 @@ func NewGrpcClient(config *Config) (*GrpcClient, error) {
 	config.GrpcOptions = append([]grpc.DialOption{
 		config.getTransportCreds(),
 		config.getAPIKeyInterceptor(),
+		config.getResponseInterceptor(),
```

Comment:
```suggestion
		config.getRateLimitInterceptor(),
```

To align with the API key interceptor.